frontend-implicaction/src/app/user/profile/components/work-experience-detail/work-experience-detail.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="row mb-2">
+  <div class="col-1">
+    <img
+      src="https://scontent.fcdg2-1.fna.fbcdn.net/v/t1.6435-9/187524910_10165049864915526_6641535558151019010_n.jpg?_nc_cat=1&ccb=1-5&_nc_sid=09cbfe&_nc_ohc=Ib_rwH5IwxMAX8CjRo6&_nc_ht=scontent.fcdg2-1.fna&oh=44394ba493a5b02dcb509b197adc252e&oe=615BBE86"
+      alt="Air France"
+      class="img-fluid">
+  </div>
+  <div class="col d-flex align-items-start flex-column align-self-center">
+    <h3 class="pb-1">{{workExperience.companyLabel}}</h3>
+    <h4 class="pt-1">{{workExperience.label}}</h4>
+  </div>
+</div>
+<div class="mb-2">
+  <p class="description-color">{{workExperience.description}}</p>
```

Comment:
je pensais plutôt à quelque chose du genre :

```suggestion
<div class="mb-2 description">
  <p>{{workExperience.description}}</p>
```
et du coup définir la règle

```css
.description p {
 color: .... 
}
```